release-content/migration-guides/zstd.md

```diff
@@ -5,3 +5,22 @@ pull_requests: [19793]
 
 A more performant zstd backend has been added for texture decompression. To enable it, disable default-features and enable feature "zstd_c".
 If you have default-features disabled and use functionality that requires zstd decompression ("tonemapping_luts" or "ktx2"), you must choose a zstd implementation with one of the following feature flags: "zstd_c" (faster) or "zstd_rust" (safer)
+
+## Migration Guide
+
+If you previously used the `zstd` feature explicitly:
+```toml
+# 0.16
+[dependencies]
+bevy = { version = "0.16", features = ["zstd"] }
+
+# 0.17 - Use the safe Rust implementation:
+[dependencies]
+bevy = { version = "0.17", features = ["zstd_rust"] }
```

Comment:
I believe this line will result in the exact same behavior as 0.16. 
If that is true, can we explicitly mention that the feature was *renamed* to this, and that by using this backend you keep the same behavior as in 0.16?